libpkg/pkg_repo_meta.c

```diff
@@ -300,6 +364,8 @@ pkg_repo_meta_default(void)
 	struct pkg_repo_meta *meta;
 
 	meta = xcalloc(1, sizeof(*meta));
+	/* Default to version 1 for now */
+	meta->version = 1;
```

Comment:
maybe we can use something like #define DEFAULT_META_VERSION on top of the file for that for clarity